.changeset/rich-goats-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-yoga/plugin-jwt': patch
+---
+
+Do not throw when \`request\` is not available in the context, it can be a WebSockets connection
```

Comment:
If this PR also adds support (and not just fixing the bug), can we add a link to the issue + a short example on how to use it? (or point to the docs?)

Reply:
How about now?